printAll.js

```diff
@@ -0,0 +1,14 @@
+const index = require("./index");
+
+function GetallColorCodedPairs() {
+  var i;
+  var colorCodePair = [];
+  for (i = 1; i < 26; i++) {
+    var colorCombo ={}
+    colorCombo.number = i;
+    colorCombo.color = index.GetColorFromPairNumber(i);
+    colorCodePair.push(colorCombo);  
+  }
+  return colorCodePair;
+}
+//console.log(GetallPairs().toString());
```

Comment:
Dont leave commented code in the submission.
BTW, this code is needed to test the above function (call is out of sync by the way). So move it to a test.